public/app/app-init.js

```
@@ -0,0 +1,613 @@
+
+// ─────────────────────────────────────────────────────────────────────────────
+//  ① Bootstrap — runs immediately on module load
+// ─────────────────────────────────────────────────────────────────────────────
+
+(async () => {
+  // 1. Auth gate — blocks until login succeeds
+  session = await requireAuth();
+  caps    = getCaps(session.tier);
+
+  // 2. Wire logout button if present
+  document.getElementById('btn-logout')
```

ESLint

[error] 'document' is not defined.

(no-undef)